src/jedi_bundle/utils/file_system.py

```diff
@@ -63,10 +63,13 @@ def check_for_executable(logger, executable):
 # --------------------------------------------------------------------------------------------------
 
 
-def subprocess_run(logger, command, abort_on_fail=True):
+def subprocess_run(logger, command, abort_on_fail=True, cwd=None):
 
     # Prepare command
-    p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    if cwd:
+        p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=cwd)
+    else:
+        p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
`cwd=None` is already the default and works as expected (run the command in the current working directory).

```suggestion
    p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=cwd)
```

For future reference, the thing I would probably recommend here is to propagate `**kwargs`. This allows `subprocess_run` to optionally use any of the other arguments to `subprocess.Popen` (except for `stdout` and `stderr`, which throw a `TypeError` that the argument is already used).

```python
def subprocess_run(logger, command, abort_on_fail=True, **kwargs):
  p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, **kwargs)